extension/page-detect.js

```diff
@@ -58,6 +60,8 @@ window.pageDetect = (() => {
 		return isRepo() && blobPattern.test(location.href);
 	};
 
+	const hasCommentForm = () => isGist() || (isRepo() && (isIssue() || isNewIssue() || (isPR() && !isPRFiles()) || isCompare()));
```

Comment:
It is *now*. That might change in the future. I just don't understand why you would want to hard code all these when you can simply check if `.js-previewable-comment-form` exists?